metric/system/process/process_linux_test.go

```diff
@@ -234,6 +234,25 @@ func TestParseProcStat(t *testing.T) {
 	assert.Equal(t, want, got, "")
 }
 
+func TestCgroupsBadCgroupsConfig(t *testing.T) {
+	_ = logp.DevelopmentSetup()
```

Comment:
optional, but since you are setting up logp, is it worthwhile to do `DevelopmentSetup(ToObserverOutput())` and then use `ObserverLogs().TakeAll()` to get the logs and make sure the error message you expect is in the log output?